src/system/SystemPacketBuffer.h

```diff
@@ -160,6 +161,8 @@ class DLL_EXPORT PacketBuffer : private pbuf
             return LWIP_MEM_ALIGN_SIZE(PBUF_POOL_BUFSIZE) - kStructureSize;
         else
             return LWIP_MEM_ALIGN_SIZE(memp_sizes[this->pool]) - kStructureSize;
+#elif CHIP_SYSTEM_PACKETBUFFER_FROM_LWIP_RAM_HEAP
+        return lwip_mem_size(this) - kStructureSize;
```

Comment:
Why shouldn't the allocated size be `this->len` or `this->tot_len`?

Reply:
Method `New(size_t aAvailableSize, uint16_t aReservedSize)` of `PacketBufferHandle` resets both of `len` and `tot_len` to zero at round line 544 in `SystemPacketBuffer.cpp`.
```c
    lPacket->len = lPacket->tot_len = 0;
```

Reply:
That is right. I overlooked the assignment to 0.